codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionHandlerTest.java

```diff
@@ -243,6 +244,17 @@ public void serverReceivingInvalidClientPrefaceStringShouldHandleException() thr
     }
 
     @Test
+    public void serverReceivingHttp1ClientPrefaceStringShouldIncludePreface() throws Exception {
+        when(connection.isServer()).thenReturn(true);
+        handler = newHandler();
+        handler.channelRead(ctx, copiedBuffer("GET /path HTTP/1.1", UTF_8));
```

Comment:
Done.  Technically the other one should be changed too since HTTP doesn't allow UTF_8 in that location.